### PR TITLE
Scrum 35 create organization config page

### PR DESCRIPTION
**Pull request by Vishnu17092003**
### Explain What was done in this change

_________________

#### PR Checklist
- [x] No lining, TS, or console errors
- [x] No commented-out code without a reason
- [x] No business logic inside UI components
- [x] Dev tests are done: all the related components are working as before

### CODE REVIEW

**Comment by Vishnu17092003** [comment]
Completed the first story

File: public/form1.png
Comment (mentorbridgeindia) [inline public/form1.png:1]: Is this needed?

File: src/pages/OrgPage.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import "./OrgPage.css";
+import { yupResolver } from "@hookform/resolvers/yup";
+import React from "react";
+import { useForm } from "react-hook-form";
+import * as yup from "yup";
+
+const schema = yup.object().shape({
+  oname: yup
```
Comment (mentorbridgeindia) [inline src/pages/OrgPage.tsx:15]: orgName

File: src/pages/OrgPage.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    .string()
+    .min(4, "Please enter a valid Organization Name")
+    .required("Organization Name is required"),
+  ecount: yup
```
Comment (mentorbridgeindia) [inline src/pages/OrgPage.tsx:19]: empCount

File: src/pages/OrgPage.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    .string()
+    .min(4, "Please enter at least 4 characters")
+    .required("Employee Count is required"),
+    web: yup
```
Comment (mentorbridgeindia) [inline src/pages/OrgPage.tsx:23]: website

**Review by mentorbridgeindia — CHANGES_REQUESTED** [review]

**Comment by mentorbridgeindia** [comment]
> ## JIRA User story link
> ### Explain What was done in this change
> #### PR Checklist
> * [x]  No lining, TS, or console errors
> * [x]  No commented-out code without a reason
> * [x]  No business logic inside UI components
> * [x]  Dev tests are done: all the related components are working as before

Where are the details?

**Pull request closed without merging**